packages/patternfly-4/react-core/src/components/BackgroundImage/BackgroundImage.js

```diff
@@ -10,26 +10,23 @@ import {
   c_background_image_BackgroundImage_2x,
   c_background_image_BackgroundImage_sm,
   c_background_image_BackgroundImage_sm_2x,
-  c_background_image_BackgroundImage_lg,
-  c_background_image_Filter
+  c_background_image_BackgroundImage_lg
 } from '@patternfly/react-tokens';
 
 export const BackgroundImageSrc = {
   xs: 'xs',
   xs2x: 'xs2x',
   sm: 'sm',
   sm2x: 'sm2x',
-  lg: 'lg',
-  filter: 'filter'
```

Comment:
@Don Do we need keep the filter here? Removing it would cause a breaking change since it is exported.